Applications/src/auto-contrast.cc

```diff
@@ -0,0 +1,104 @@
+/*=========================================================================
+
+  Library   : Image Registration Toolkit (IRTK)
+  Module    : $Id$
+  Copyright : Imperial College, Department of Computing
+              Visual Information Processing (VIP), 2008 onwards
+  Date      : $Date$
+  Version   : $Revision$
+  Changes   : $Author$
+Copyright (c) 1999-2014 and onwards, Imperial College London
+All rights reserved.
+See LICENSE for details
+
+=========================================================================*/
+
+#include "mirtk/Common.h"
+#include "mirtk/Options.h"
+
+#include "mirtk/ImageReader.h"
+#include "mirtk/IOConfig.h"
+#include "mirtk/GenericImage.h"
+
+using namespace mirtk;
+
+char *output_name = NULL;
+
+
+void PrintHelp(const char *name)
+{
+  cout << endl;
+  cout << "Usage: " << name << " <image> <output> [options]" << endl;
+  cout << endl;
+  cout << "Description:" << endl;
+  cout << "  This program auto contrast an image." << endl;
+  cout << endl;
+  cout << "Arguments:" << endl;
+  cout << "  input    Input image." << endl;
+  cout << "  output   Contrasted output image." << endl;
+  cout << endl;
+  cout << "Options:" << endl;
+  cout << "  -Tp <value>        Target padding value all pixel < value = value" << endl;
```

Comment:
Could probably say the program clamps the input image, where the default thresholds used are set to `mean +/- 3 * sdev`? And because clamping with fixed thresholds is already possible with `calculate-element-wise`, it would make more sense to have the multiplicative factor of 3 to be an option you can specify. Instead of (or in addition to) upper and lower thresholds.